docs/install/airgapped-clusters.md

```diff
@@ -0,0 +1,365 @@
+---
+title: "Airgapped installation"
+weight: 50
+---
+
+> The following page is for Advanced users. The full procedure is estimated to
+> take ~60 minutes to complete.
+
+Clusters without access to the internet require you to explicitly specify the 
+resources to be installed. To install Ondat on an airgapped cluster, you need to do the following:
+
+- Pull OCI images to private Docker registries.
+- Retrieve and amend YAML for the Ondat cluster-operator and CRDs.
+- Define the StorageOSCluster CustomResource YAML.
+- (If applicable) Retrieve and amend YAML for Etcd operator and Etcd CustomResource.
+
+## Step 1. Pulling images into a private registry:
+
+There are the following sets of images to pull:
+* the Ondat operator image
+* the images in the StorageOSCluster definition that define the images for each component of the cluster 
+* (if applicable) the images to run Etcd as Pods using the Etcd operator deployed by Ondat
+
+1. Install the Ondat operator: `storageos/operator:v2.5.0` and `quay.io/brancz/kube-rbac-proxy:v0.10.0`
```

Comment:
Does it mean we need to change the operator version here every time we make a new release?

Reply:
Yes, before the simplification of the docs to a less "hugo-ish" like style, we had this parametrised in one place, but now we can't and we need to apply changes in 100s of places every time we change the version of any component we release.

Reply:
this is terrifying, I think we need to make sure we do a sweep before we make a new release...